images/gatk-sv_issue_661/compute_AFs.py

```diff
@@ -164,17 +188,46 @@ def calc_allele_freq(record, samples, prefix=None, hemi=False):
         n_alt_count_2 = 0
         n_gts_with_gt_0_alts = 0  # Used specifically for hemizygous sites
         for GT in GTs:
-            AN += len([allele for allele in GT if allele !=
-                       '.' and allele is not None])
-            AC += len([allele for allele in GT if allele !=
-                       '.' and allele != 0 and allele is not None])
+            AN += len([allele for allele in GT if allele != '.' and allele is not None])
+            AC += len(
+                [
+                    allele
+                    for allele in GT
+                    if allele != '.' and allele != 0 and allele is not None
+                ]
+            )
             if GT == (0, 0):
                 n_alt_count_0 += 1
-            if len([allele for allele in GT if allele == 0 and allele != '.' and allele is not None]) == 1 \
-                    and len([allele for allele in GT if allele != 0 and allele != '.' and allele is not None]) == 1:
+            if (
+                len(
+                    [
+                        allele
+                        for allele in GT
+                        if allele == 0 and allele != '.' and allele is not None
+                    ]
+                )
+                == 1
+                and len(
+                    [
+                        allele
+                        for allele in GT
+                        if allele != 0 and allele != '.' and allele is not None
+                    ]
+                )
+                == 1
+            ):
```

Comment:
Always cracks me up that the machine thinks this is easier to read…

But the real fix is to make `line_len` bigger, and that's a bigger change for another day.